### PR TITLE
added RA posting for synthetic satellite imaging

### PR DESCRIPTION
**Pull request by mbmcgarry**

### CODE REVIEW

File: source/jobs/ra-2016-imaging-cvt.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+* Research appropriate parameters and specifications to make simplified but
+  representative fake satellite images (resolution, image size, variety of
+  information)
+* Define the properties of relevant image features (such as truck shape/size,
+  etc)
+* Write a program in Python that uses a table of features and positions to
+  create fake images. Program will be flexible/extensible for additional
+  features in the future.
+
+To be considered for an hourly Research Assistant position (50% plus tuition
```
Comment (gonuke) [inline source/jobs/ra-2016-imaging-cvt.rst]: not hourly.  RA or PA

File: source/jobs/ra-2016-imaging-cvt.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+remission), please send a letter of interest, resume, and unofficial
+transcript to wilsonp _at_ engr.wisc.edu.  For further questions please
```
Comment (gonuke) [inline source/jobs/ra-2016-imaging-cvt.rst]: Maybe we can use the address: cnerg-cvt@lists.wisc.edu   
I'm pretty sure it works...

File: source/jobs/ra-2016-imaging-cvt.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+contact either wilsonp _at_ engr.wisc.edu or mbmcgarry _at_ wisc.edu.
+
```
Comment (gonuke) [inline source/jobs/ra-2016-imaging-cvt.rst]: What did we say last time about citizenship?

**Comment by mbmcgarry** [comment]
CVT rules say students must be US citizens (but not post-docs).

**Comment by gonuke** [comment]
But did we include something about citizenship in the posting?

File: source/jobs/ra-2016-imaging-cvt.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  information)
+* Define the properties of relevant image features (such as truck shape/size,
+  etc)
+* Write a program in Python that uses a table of features and positions to
+  create fake images. Program will be flexible/extensible for additional
+  features in the future.
+
+To be considered for a Research Assistant position (50% plus tuition
+remission), please send a letter of interest, resume, and unofficial
+transcript to  cnerg-cvt _at_ lists.wisc.edu
```
Comment (gonuke) [inline source/jobs/ra-2016-imaging-cvt.rst]: Add yours and/or my name, but still this email address

File: source/jobs/ra-2016-imaging-cvt.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+<http://fuelcycle.org>`_  to model current and next-generation nuclear fuel
+cycles as part of the `Consortium for Verification Technology
+<http://cvt.engin.umich.edu/>`_. |Cyclus| is an agent-based, systems-level
+simulator that tracks discrete material flow through the entire fuel cycle,
+from mining through burnup in reactors to a repository.  
+
+We seek a research assistant to develop synthetic satellite images of nuclear
+facilities for monitoring and verification.  Satellite images can provide
+information on material shipments, production and maintenance staffing, physical
+ and environmental impact, etc.  Synthetic images will be designed with
```
Comment (gonuke) [inline source/jobs/ra-2016-imaging-cvt.rst]: spaces at beginning of line - rendering??

File: source/jobs/ra-2016-imaging-cvt.rst [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+ changeable features at different times that are correlated with other fuel
+ cycle signals.  These images will be used in Cyclus to study the clandestine
+ material diversion from declared fuel cycles.  Due to funding restrictions,
+ only US citizens can be considered.
+
+Experience
+----------
+
+* Masters or PhD student in Nuclear Engineering, Computer Science, Physics,
```
Comment (gonuke) [inline source/jobs/ra-2016-imaging-cvt.rst]: other fields? Astronomy? AOS? Geography?

File: source/jobs/ra-2016-imaging-cvt.rst [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+  Math or related field
+* Interest in international security and nuclear non-proliferation OR
+* Experience with image processing and analysis
+* No programming experience required, but must be willing to learn
```
Comment (gonuke) [inline source/jobs/ra-2016-imaging-cvt.rst]: some experience desired....

**Comment by mbmcgarry** [comment]
@gonuke : updated with your suggestions

**Pull request merged**